tools/packaging/common/istio-start.sh

```diff
@@ -66,7 +66,7 @@ if [ "${ISTIO_CUSTOM_IP_TABLES}" != "true" ] ; then
     if [[ ${1-} == "init" || ${1-} == "-p" ]] ; then
```

Comment:
I really can't find where this codepath is used but I adjusted it anyway. Perhaps it is something old we don't need to support anymore??